srcDoc/command_line_parameters.rst

```diff
@@ -439,6 +439,25 @@ Test Parameters for Publishing and Subscribing Applications
 
    **Default:** ``Not enabled``
 
+- ``-outputFormat <format>``
+
+   Specify the format of the printed data to facilitate display or export.
```

Comment:
Specify the format for the printed data to facilitate its display or to export it.

Reply:
ok